src/TodoList.tsx

```diff
@@ -0,0 +1,40 @@
+// [TO IMPLEMENT] 
```

Comment:
TypeScript is commonly used when developers want to add static typing to their JavaScript code. It provides a way to catch type-related errors early in the development process, making the code easier to maintain. 

Why are you writing your assignments in Typescript? If this is your first time learning React, the best thing to do is learn with a language you are most comfortable with (I'm assuming Javascript).